src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -681,3 +682,21 @@ macro_rules! metric {
         client.track_metric($name.into(), $value);
     }};
 }
+
+#[cfg(test)]
+mod tests {
+    use crate as onefuzz_telemetry;
```

Comment:
I didn't `use super::*;` here because the crate name needs to be `onefuzz_telemetry` in order for the macro to expand properly.